modules/apps/commerce/commerce-fragment/commerce-fragment-collection-contributor/commerce-fragment-collection-contributor-cart/src/main/resources/com/liferay/commerce/fragment/collection/contributor/cart/dependencies/mini-cart/index.html

```diff
@@ -0,0 +1,13 @@
+<div class="mini-cart-fragment ${fragmentEntryLinkNamespace}">
+	<style>
+		.${fragmentEntryLinkNamespace} {
```

Comment:
You may ruin intro troubles here is the namespace start with a number, which I believe it may happen

Instead of having the style tag directly in the body of the fragment, we can simple add a inline style in the div above

Something like:

```
<div class="mini-cart-fragment [#if configuration.iconColor?has_content]style="--mini-cart-icon-color: ${configuration.iconColor};"[/#if]>
```